docs/code_style.md

```diff
@@ -255,6 +255,153 @@ Sometimes we use `std::unique_ptr/smart_ptr` to delay initialization or for opti
 
 Use a default-constructed `std::unique_ptr/smart_ptr` where null is expected.
 
+### P19 name prefix for factory functions
+
+Use `make` name/prefix for factory functions instead of `build` or `create`:
+
+	SentryPtrPair make_sentry(...)
+	std::shared_ptr<TimerImpl> make(...)
+	std::string make_thread_name(...)
+
+`build` prefix can be used for a “builder” pattern, where it might have no parameters, and potential side effects, for example `RecSplit::build()`.
+
+### P20 to_string method naming
+
+Use `to_string()` by default to convert an object to std::string (e.g. for logging purposes). Only provide `operator<<` if needed (implemented on top of `to_string()`).
+
+If a global function is required, `xxx_to_string()` name should be used.
+
+### P21 multiline parameters indentation
+
+Default to 1 parameter per line.
+
+Accepted styles:
+
+	ValidationResult execute_block(
```

Comment:
Does clang-format is happy with both variants?

I think you can configure clang-format to have desired formatting but this likely requires putting the line length limit.

Reply:
Yes, this is the status quo.
I wanted to be stricter here, but we ended up fixating the status quo, because nobody was sure how to reconfigure the clang-format. And in particular, people wanted to have a clear separation of the function signature from the body. E.g. in Go/Swift/Rust style it is possible to have the final `) {` on a separate line, and even have comma after the last argument.